temboardui/static/js/home.js

```diff
@@ -103,10 +110,13 @@ $(function() {
     var checks = instance.checks;
     var value = 0;
     if (checks.CRITICAL) {
-      value += checks.CRITICAL * 1000;
+      value += checks.CRITICAL * 1000000;
```

Comment:
Why `1000000` now ?

Reply:
It's to make sure that instances with critical checks will appear first.
There may be better ways to prioritize checks.
It could also be critical first, then no difference for the warning or undef.